README.md

```diff
@@ -1,5 +1,20 @@
 # lineup-refinery-docker
-Builds a Refinery visualization Docker image for Lineup
+This builds a Refinery visualization Docker image for Lineup.
+
+For a preview, you can run the Docker container locally with your own data:
+```
+docker run \
+      --detach \
+      --publish 8888:80 \
+      --volume /my/local/data.csv:/tmp/1.csv \
+      mccalluc/lineup_refinery
```

Comment:
Would this need another line break?

Reply:
It could all be on one line, but since bash doesn't have any syntax to delimit kwarg pairs, for a script like this I'd lean towards more lines here. Does that sound ok?